src/main/scala/org/broadinstitute/dsde/vault/services/analysis/UpdateServiceHandler.scala

```diff
@@ -94,7 +94,7 @@ case class UpdateServiceHandler(requestContext: RequestContext, dmService: Actor
         case true =>
           if (fileCount == bossObjects.size) {
             log.debug("'X-Force-Location' Analysis update complete")
-            requestContext.complete(new Analysis(dmId.get, analysis.input, analysis.metadata, analysis.files))
+            requestContext.complete(new Analysis(dmId.get, analysis.input, analysis.metadata, Option(providedFiles)))
```

Comment:
Is this actually necessary? If it's not, I'd prefer to leave it as-is, so we're returning what DM returned.